packages/config/src/models/question.ts

```diff
@@ -0,0 +1,71 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import { Document, model, Schema } from 'mongoose'
+
+export enum FieldType {
+  TEXT = 'TEXT',
+  TEL = 'TEL',
+  NUMBER = 'NUMBER',
+  TEXTAREA = 'TEXTAREA',
+  SUBSECTION = 'SUBSECTION',
+  PARAGRAPH = 'PARAGRAPH'
+}
+
+export const validFieldType = Object.values(FieldType)
+
+export interface IMessageDescriptor {
+  id: string
+  description: string
+  defaultMessage: string
+}
+
+export interface IQuestion {
+  fieldId: string
+  label?: IMessageDescriptor
+  placeholder?: IMessageDescriptor
+  maxLength?: number
+  fieldName?: string
+  fieldType?: FieldType
+  preceedingFieldId?: string
+  required?: boolean
+  enabled: boolean
+  custom?: boolean
+  initialValue?: string
+}
+
```

Comment:
The config database will store new custom fields, plus any field whose vertical position has changed, and any field that has been enabled or disabled.  My plan is, that the form object will first be created in the reducer using the registerForms prop, then for each field that is loaded from this database, I will render the custom prop in the section using the dot notated fieldId as we have planned: e.g. "birth.mother.date-of-birth-of-pet-dog" and then the vertical position informed using preceedingFieldId.  Wherever preceedingFieldId is positioned vertically, the custom field will go after it.  I felt this was easier than using an index number to code.  It will also hide or show any of the default fields using the enabled prop